### ISSUE TITLE
Can't login to aws instance after changing the console

### ISSUE THREAD

**Issue by mariomarin** [rancherio/os#132]
First login

```
ssh rancherhost
Warning: Permanently added 'secret ip' (ECDSA) to the list of known hosts.
```

Install *ubuntu-console* and reboot

```
[rancher@rancher ~]$ sudo rancherctl addon list                 
ubuntu-console disabled
[rancher@rancher ~]$ sudo rancherctl addon enable ubuntu-console
[rancher@rancher ~]$ sudo rancherctl addon list                 
ubuntu-console enabled
[rancher@rancher ~]$ sudo reboot
Connection to 10.1.xxx.xxx closed by remote host.
Connection to 10.1.xxx.xxx closed.
```

I tried to login again and it failed

```
ssh rancherhost
ssh_exchange_identification: Connection closed by remote host
```

**Comment by cloudnautique** [comment]
Just to confirm, you are logging in as the `rancher` user after the reboot?

I tested on a t1.micro instance in us-west-1, and thought I reproduced, but it just took around 1-2 mins to come back. The ubuntu-console image is not part of the base os and has to be pulled from the docker hub registry on the first boot with it enabled. The SSH server is a part of that image, and the box seems dead for a while. Did you try immediately? or did you give it some time?

**Comment by mariomarin** [comment]
I gave it some time. I'll try again.

**Comment by cloudnautique** [comment]
@mariomarin If you are able to reproduce, can you let me know which AMI/Instance type? Also, if you wouldn't mind can you copy the console logs?

Thanks

**Comment by wlan0** [comment]
ping @mariomarin -- any updates on this issue? I tried this with rancheros-v0.3.0-1 - ami-ca3e39a2

I cannot reproduce this issue.

**Comment by deniseschannon** [comment]
with v0.3.0, this should be fixed. Please re-open if you are still having issues.

**Comment by willseward** [comment]
I'm having the same issue. It can't download the ubuntu-console image.

```
[    0.000000] Command line: BOOT_IMAGE=/boot/vmlinuz-v0.3.3-rancheros console=tty0 console=ttyS0
```

```
[34mINFO[0m[0002] Launching System Docker                      
[34mINFO[0m[0000] Waiting for Docker at unix:///var/run/system-docker.sock 
[   36.147977] nf_conntrack version 0.5.0 (7983 buckets, 31932 max)
[   36.151525] Bridge firewalling registered
[   36.155168] ip_tables: (C) 2000-2006 Netfilter Core Team
[34mINFO[0m[0000] Connected to Docker at unix:///var/run/system-docker.sock 
[34mINFO[0m[0000] Loading images from /images.tar              
[34mINFO[0m[0001] Done loading images from /images.tar         
[34mINFO[0m[0001] [acpid]: Implicitly linked to 'syslog'       
[34mINFO[0m[0001] [cloud-init]: Implicitly linked to 'syslog'  
[34mINFO[0m[0001] [ntp]: Implicitly linked to 'syslog'         
[34mINFO[0m[0001] [network]: Implicitly linked to 'syslog'     
[34mINFO[0m[0001] [cloud-init-pre]: Implicitly linked to 'syslog' 
[34mINFO[0m[0001] [docker]: Implicitly linked to 'syslog'      
[34mINFO[0m[0001] [preload-system-images]: Implicitly linked to 'syslog' 
[34mINFO[0m[0001] [udev]: Implicitly linked to 'syslog'        
[34mINFO[0m[0001] [console]: Implicitly linked to 'syslog'     
[34mINFO[0m[0001] [dockerwait]: Implicitly linked to 'syslog'  
[34mINFO[0m[0001] [preload-user-images]: Implicitly linked to 'syslog' 
[   43.701085] IPv6: ADDRCONF(NETDEV_UP): veth78b3279: link is not ready
[   43.704685] IPv6: ADDRCONF(NETDEV_CHANGE): veth78b3279: link becomes ready
[   43.708363] device veth78b3279 entered promiscuous mode
[   43.711218] docker-sys: port 1(veth78b3279) entered forwarding state
[   43.714624] docker-sys: port 1(veth78b3279) entered forwarding state
[   43.771403] eth0: renamed from veth7a9b229
[   43.848552] udevd[1]: starting version 2.1.1
[   43.858488] veth7a9b229: renamed from eth0
[   43.868703] docker-sys: port 1(veth78b3279) entered disabled state
[   43.872224] device veth78b3279 left promiscuous mode
[   43.874973] docker-sys: port 1(veth78b3279) entered disabled state
[   43.981173] 9p: Installing v9fs 9p2000 file system support
[   43.984392] 9pnet: Could not find request transport: virtio
[34mINFO[0m[0007] [console]: Implicitly linked to 'syslog'     
[34mINFO[0m[0007] [cloud-init]: Implicitly linked to 'syslog'  
[34mINFO[0m[0007] [dockerwait]: Implicitly linked to 'syslog'  
[34mINFO[0m[0007] [network]: Implicitly linked to 'syslog'     
[34mINFO[0m[0007] [docker]: Implicitly linked to 'syslog'      
[34mINFO[0m[0007] [ntp]: Implicitly linked to 'syslog'         
[34mINFO[0m[0007] [preload-user-images]: Implicitly linked to 'syslog' 
[   45.980014] IPv6: ADDRCONF(NETDEV_UP): veth59be121: link is not ready
[   45.983601] IPv6: ADDRCONF(NETDEV_CHANGE): veth59be121: link becomes ready
[   45.987168] device veth59be121 entered promiscuous mode
[   45.990011] docker-sys: port 1(veth59be121) entered forwarding state
[   45.993198] docker-sys: port 1(veth59be121) entered forwarding state
[   46.015379] eth0: renamed from veth98eba0c
[   46.070459] veth98eba0c: renamed from eth0
[   46.080227] docker-sys: port 1(veth59be121) entered disabled state
[   46.083557] device veth59be121 left promiscuous mode
[   46.086145] docker-sys: port 1(veth59be121) entered disabled state
[   46.178731] 9pnet: Could not find request transport: virtio
[31mERRO[0m[0030] Failed to load ubuntu-console : Get https://raw.githubusercontent.com/rancherio/os-services/v0.3.3/u/ubuntu-console.yml: dial tcp: lookup raw.githubusercontent.com on 8.8.4.4:53: read udp 8.8.4.4:53: i/o timeout 
[34mINFO[0m[0030] [ntp]: Implicitly linked to 'syslog'         
[34mINFO[0m[0030] [console]: Implicitly linked to 'syslog'     
[34mINFO[0m[0033] RancherOS v0.3.3 started      
```

**Comment by willseward** [comment]
This looks like another version of the same error.

```
[34mINFO[0m[0024] GET /containers/console/json                 
[31mERRO[0m[0024] Handler for GET /containers/{name:.*}/json returned error: no such id: console 
[31mERRO[0m[0024] HTTP Error                                    [31merr[0m=no such id: console [31mstatusCode[0m=404
```

**Comment by deniseschannon** [comment]
@willseward Can you confirm how you launched RancherOS? If I launch RancherOS through the AWS AMI, it starts with the ubuntu-console. It might make sense to make a new issue as the original issue was related to RancherOS on AWS.

Can you try a `docker pull` on any image to confirm if networking is working? Based on the logs, it looks like you have a networking issue with RancherOS.

**Comment by Oskoss** [comment]
@willseward Were you able to find a solution to this? 

Thanks....

**Comment by blandes** [comment]
I have an instance with the same issue. I have RancherOS installed and enabled ubuntu-console through services, but now I cannot ssh into the server. I've tried rancher, ubuntnu, and root but no dice. Anything on this?

**Comment by ls-amine** [comment]
Can any one advise if there is a fix for this issue.
thanks in advance.

**Comment by willseward** [comment]
Oh my... I lost track of this. 

@ls-amine I can't advise, sorry.

**Comment by joshwget** [comment]
@ls-amine This is a pretty old issue. If you're still experiencing something similar can you open a new issue?

**Comment by SvenDowideit** [comment]
We did just fix an issue like this in 0.8.0 rc6

On 26 Jan 2017 7:31 AM, "Josh Curl" <notifications@github.com> wrote:

@ls-amine <https://github.com/ls-amine> This is a pretty old issue. If
you're still experiencing something similar can you open a new issue?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/rancher/os/issues/132#issuecomment-275239099>, or mute
the thread
